lib/form-utils.js

```diff
@@ -71,6 +72,15 @@ export const verifyChecked = (errors, data, field) => {
   return errors;
 };
 
+export const verifyEmailPattern = (errors, data, field) => {
```

Comment:
Nice :+1:

Reply:
@Betree For `StyledInputFormikField`, had already used `StyledInputField` in the form before realizing it existed and much easier to use, so I changed all the inputs to use it. My only feedback is we should add it to the Style Guide and explain why we should favour its usage over `StyledInputField`.